components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/persistence/DBConnectionException.java

```diff
@@ -0,0 +1,47 @@
+/*
+  * Copyright (c) 2023, WSO2 Inc. (http://www.wso2.com).
```

Comment:
WSO2 LLC

Reply:
update license header in https://github.com/wso2/carbon-identity-framework/pull/4416/commits/b396ddc28e3b89ac8acc6b62c712ee81161866ff